### ISSUE TITLE
Bitbucket Server ALM configuration fails (Http 404 for /api/alm_integrations/check_pat?almSetting=bitbucket  && /api/alm_integrations/set_pat)

### ISSUE THREAD

**Issue by benze** [mc1arke/sonarqube-community-branch-plugin#304]
I'm trying to run SonarQube 8.6.1 Community Edition Docker container with the 1.7.0-SNAPSHOT plugin from https://github.com/mc1arke/sonarqube-community-branch-plugin/pull/301#issue-564436289.  However, when I try to configure a Bitbucket server, the configuration fails as the following API endpoints are not found:

```
HTTP Status 404 : /api/alm_integrations/check_pat?almSetting=bitbucket
HTTP Status 404 : /api/alm_integrations/set_pat
```

I tried to roll back to SonarQube 8.5-community docker container with the 1.6.0.jar plugin, but with the same error message.

Is there a special setting/configuration that I am missing in order to enable Bitbucket ALM integration with this plugin? 

*Steps to reproduce:*
- spin up container
```
docker-compose.yml
  sonar: 
    image: sonarqube:8.5-community
    ports:
      - "9000:9000"
    environment:
      TZ: America/Toronto
    volumes:
      - "./conf:/opt/sonarqube/conf"
      - "./logs:/opt/sonarqube/logs"
      - ./plugins/sonarqube-community-branch-plugin-1.6.0.jar:/opt/sonarqube/extensions/plugins/sonarqube-community-branch-plugin.jar
      - ./plugins/sonarqube-community-branch-plugin-1.6.0.jar:/opt/sonarqube/lib/common/sonarqube-community-branch-plugin.jar
```
- open network inspection tool in browser
- login as admin
- administration/ALM Integrations.  Configure a Bitbucket Server.  Add PAT token and save.
- click the `+` symbol, select Bitbucket server
- Try to enter PAT token.
- 404 visible for /api/alm_integrations/check_pat & /api/alm_integrations/set_pat endpoints

![image](https://user-images.githubusercontent.com/2797306/106693565-bf181300-65a4-11eb-9aa7-bb5da2e48cbb.png)

Unable to connect/authenticate to a Bitbucket server.

This occurs in both Sonar 8.5 and 8.6.  Has anyone been able to successfully connect/enable BB ALM integration?

**Comment by devstructor** [comment]
I have the same issue with gitlab integration

**Comment by winterheart** [comment]
Hi.
You can try test 1.7.0 (not merget yet) from #301.

**Comment by mc1arke** [comment]
There are various new endpoints that have been introduced in recent Sonarqube versions, or that are now being called by the Front-end where they weren't previously required. I've not had a chance to go through and implement them since the 8.5 release was published that started depending on them. See https://next.sonarqube.com/sonarqube/web_api/api/alm_integrations?internal=true for some of these endpoints and what they do.

**Comment by mc1arke** [comment]
Fixed in 1.7.0 of the plugin

**Comment by lucasoares** [comment]
@mc1arke is 1.7.0 compatible with sonarqube 8.6? Because in the release notes it says it is not.

Which version is compatible with 8.6 then?
